### PR TITLE
Add Cross-Contract Test to Fungible Token Example Test Cases

### PR DESCRIPTION
**Pull request by idea404**
This PR adds the missing test marked with a `// TODO` for `fungible-token`

### CODE REVIEW

File: examples/package.json [hunk 1]
```diff
@@ -33,7 +33,8 @@
   "license": "Apache-2.0",
   "dependencies": {
     "lodash-es": "^4.17.21",
-    "near-sdk-js": "file:../"
+    "near-sdk-js": "file:../",
+    "yargs": "^17.5.1"
```
Comment (volovyk-s) [inline examples/package.json:37]: Looks like this change should not be here.

**Review by volovyk-s — COMMENTED** [review]
Great! Just one nit.

File: examples/package.json (hunk 1 above)
Comment (idea404) [reply examples/package.json]: Right! Have removed, ty.

**Review by volovyk-s — APPROVED** [review]
Please, fix the small conflict and feel free to merge this PR. Great work!

**Pull request merged**